warp-tori/src/main/java/com/schibsted/nmp/warp/brands/tori/ToriDarkColors.kt

```diff
@@ -197,6 +198,12 @@ internal object ToriComponentDarkColors : WarpComponentColors {
     override val navBar: WarpNavBarColors = ToriNavBarDarkColors
     override val tooltip: WarpTooltipColors = ToriTooltipDarkColors
     override val switch: WarpSwitchColors = ToriSwitchDarkColors
+    override val spinner: WarpSpinnerColors = ToriSpinnerDarkColors
+}
+
+private object ToriSpinnerDarkColors : WarpSpinnerColors {
```

Comment:
No need to create an object for these as the colors used are not custom component tokens, but already existing semantic color tokens. The spinner colors are defined in https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit?gid=1588196525#gid=1588196525